README.rst

```diff
@@ -13,1837 +13,30 @@ Turn complex GraphQL queries into optimized database queries.
--  `Directives <#directives>`__
-
-   -  `@optional <#optional>`__
-   -  `@output <#output>`__
-   -  `@fold <#fold>`__
-   -  `@tag <#tag>`__
-   -  `@filter <#filter>`__
-   -  `@recurse <#recurse>`__
-   -  `@output\_source <#output-source>`__
-
--  `Supported filtering operations <#supported-filtering-operations>`__
-
-   -  `Comparison operators <#comparison-operators>`__
-   -  `name\_or\_alias <#name-or-alias>`__
-   -  `between <#between>`__
-   -  `in\_collection <#in-collection>`__
-   -  `not\_in\_collection <#not-in-collection>`__
-   -  `has\_substring <#has-substring>`__
-   -  `starts\_with <#starts-with>`__
-   -  `ends\_with <#ends-with>`__
-   -  `contains <#contains>`__
-   -  `not\_contains <#not-contains>`__
-   -  `intersects <#intersects>`__
-   -  `has\_edge\_degree <#has-edge-degree>`__
-   -  `is\_null <#is-null>`__
-   -  `is\_not\_null <#is-not-null>`__
-
--  `Type coercions <#type-coercions>`__
--  `Meta fields <#meta-fields>`__
-
-   -  `\_\_typename <#typename>`__
-   -  `\_x\_count <#x-count>`__
-
--  `The GraphQL schema <#the-graphql-schema>`__
--  `Execution model <#execution-model>`__
--  `SQL <#sql>`__
-
-   -  `End-To-End SQL Example <#end-to-end-sql-example>`__
-   -  `Advanced Features <#advanced-features>`__
-
--  `Macro system <#macro-system>`__
-
-   -  `Macro registry <#macro-registry>`__
-   -  `Schema for defining macros <#schema-for-defining-macros>`__
-   -  `Macro edges <#macro-edges>`__
-
--  `Miscellaneous <#miscellaneous>`__
-
-   -  `Pretty-Printing GraphQL
-      Queries <#pretty-printing-graphql-queries>`__
-   -  `Expanding @optional vertex
-      fields <#expanding-optional-vertex-fields>`__
-   -  `Optional type_equivalence_hints compilation
-      parameter <#optional-type_equivalence_hints-parameter>`__
-   -  `SchemaGraph <#schemagraph>`__
-   -  `Cypher query parameters <#cypher-query-parameters>`__
-
--  `FAQ <#faq>`__
--  `License <#license>`__
-
-Features
---------
-
--  **Databases and Query Languages:** We currently support a single
-   database, OrientDB version 2.2.28+, and two query languages that
-   OrientDB supports: the OrientDB dialect of gremlin, and OrientDB's
-   own custom SQL-like query language that we refer to as MATCH, after
-   the name of its graph traversal operator. With OrientDB, MATCH should
-   be the preferred choice for most users, since it tends to run faster
-   than gremlin, and has other desirable properties. See the Execution
-   model section for more details.
-
-   Support for relational databases including PostgreSQL, MySQL, SQLite,
-   and Microsoft SQL Server is a work in progress. A subset of compiler
-   features are available for these databases. See the `SQL <#sql>`__
-   section for more details.
-
--  **GraphQL Language Features:** We prioritized and implemented a subset of all functionality
-   supported by the GraphQL language. We hope to add more functionality over time.
-
-End-to-End Example
-------------------
```

Comment:
I changed this so that you can see the end-to-end example fully without a link to the Getting Started page since we can simply include the end-to-end OrientDB example I just added to the README.rst.